app/services/prisoner_contact_list.rb

```diff
@@ -0,0 +1,29 @@
+class PrisonerContactList
+  delegate :approved, to: :contact_list
```

Comment:
that's a bit convoluted. I was looking for that method missed this lines a few times.